sap/adt/wb.py

```diff
@@ -301,17 +302,20 @@ def try_activate(adt_object):
     if resp.text:
         Marshal.deserialize(resp.text, results)
 
+    # fetch object to refresh object attributes (e.g. current activation status)
+    adt_object.fetch()
```

Comment:
@jfilak This is the most important change -> fetch (GET) after activation (POST)